FlowCryptAppTests/Functionallity/OrganisationalRulesTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  OrganisationalRulesTests.swift
+//  FlowCryptAppTests
+//
+//  Created by Anton Kharchevskyi on 17.09.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import FlowCrypt
+
+class OrganisationalRulesTests: XCTestCase {
+    var sut: OrganisationalRules! {
+        .init(clientConfiguration: clientConfiguration)
+    }
+    var clientConfiguration: ClientConfiguration!
+
+    func testIsUsingKeyManagerURL() {
+        clientConfiguration = ClientConfiguration(keyManagerUrl: "https://test.ekm.flowcrypt.com")
+        XCTAssertTrue(sut.isUsingKeyManager)
+
+        clientConfiguration = ClientConfiguration(keyManagerUrl: nil)
+        XCTAssertFalse(sut.isKeyManagerUrlValid)
+    }
+
+    func testIsUsingValidKeyManagerURL() {
+        // valid url check in
+        clientConfiguration = ClientConfiguration(keyManagerUrl: "")
+        XCTAssertFalse(sut.isKeyManagerUrlValid)
+
+        clientConfiguration = ClientConfiguration(keyManagerUrl: "not a url string")
+        XCTAssertFalse(sut.isKeyManagerUrlValid)
+    }
+
+    func testMustAutoImportOrAutogenPrvWithKeyManager() {
+        clientConfiguration = ClientConfiguration(
+            flags: [.privateKeyAutoimportOrAutogen],
+            keyManagerUrl: "https://test.ekm.flowcrypt.com"
```

Comment:
Please use some URL that's obvious it's not a real URL like `https://ekm.example.com` in all of these tests.

Then it's obvious which ones may actually get called during tests and which not.